packages/react-chat/src/components/Button/Primary.ts

```diff
@@ -7,6 +7,11 @@ export const PrimaryButton = styled(tag(Container, ButtonVariant.PRIMARY), {
   height: '$md',
   color: '$white',
   trans: ['background-color'],
+  textOverflow: 'ellipsis',
+  overflow: 'hidden',
+  padding: '10px',
```

Comment:
was the original ask from the ticket for it to wrap around?

Reply:
synced with Mike, he approved on this option